src/crsAdapter/TomaAdapter.js

```diff
@@ -305,6 +313,33 @@ class TomaAdapter {
      * @private
      * @param xml object
      * @param lineNumber number
+     * @returns {object}
+     */
+    mapRoundTripServiceFromXmlObjectToAdapterObject(xml, lineNumber) {
+        let service = {
+            type: SERVICE_TYPES.roundTrip,
+            bookingId: xml['ServiceCode.' + lineNumber],
+            destination: xml['Accommodation.' + lineNumber],
+            no: xml['Count.' + lineNumber],
+            startDate: moment(xml['From.' + lineNumber], CONFIG.crs.dateFormat).format(this.options.useDateFormat),
+            endDate: moment(xml['To.' + lineNumber], CONFIG.crs.dateFormat).format(this.options.useDateFormat),
+            title: xml['Title.' + lineNumber],
```

Comment:
to be correct `xml['Title.' + lineNumber]` is not really a title - it is the "salutation". title is something like "dr." or "M.Sc."